.github/workflows/ci.yml

```diff
@@ -60,7 +60,8 @@ jobs:
           # github.ref_name - The branch or tag name that triggered the workflow run - same as github.ref though without the leading refs/[heads|tags]/
           echo "github.ref_name: $GITHUB_REF_NAME"
           # gitbub.repository - The owner and repository name. For example, Codertocat/Hello-World
-          echo "$github.repository: $GITHUB_REPOSITORY"
+          # On forked repos, this will show <account_of_fork>/<repo>
```

Comment:
Does this mean for PRs as well? If not, what situations will this be run from a forked repo?

Reply:
I'm not actually sure, once we've merged the framework ci.yml we can do a PR to it and see what comes up in context.  I'm going to assume it'll be silverstripe, if not and it shows the fork, meaning the action is running on the fork and not the base, it's not the end of the world, it just means that codecov won't run on the pull-request, though it will run post merge on the `4` / `4.11` branch